include/odp/api/spec/crypto.h

```diff
@@ -611,6 +611,17 @@ typedef struct odp_crypto_session_param_t {
 
 		/** Cipher Initialization Vector (IV) */
 		odp_crypto_iv_t cipher_iv;
+
+		/** Cipher IV length */
+		struct {
+			/** Unused padding field */
+			uint8_t *dummy_padding_0;
```

Comment:
Comment could mention that intention is to remove this field when iv and cipher_iv above are removed.

Reply:
Agreed not to change the comment since the last API patch makes the field part of the deprecated API (i.e. not normally visible).